app/src/main/java/com/android/partagix/model/UserViewModel.kt

```diff
@@ -61,6 +61,10 @@ class UserViewModel(
         )
   }
 
+  fun setUser(user: User) {
+    updateUIState(user)
+  }
```

Comment:
To easily improve coverage, you can add a test for this function (should be very easy to test)